### PR TITLE
Version 0.3.2

### PR DESCRIPTION
**Pull request by pixelstrolch**
@grappler could you please review again and check how I solved the issues from #29.

### CODE REVIEW

File: inc/extras-penguin-gold.php [hunk 1]
```diff
@@ -54,15 +54,10 @@ function penguin_stylesheet_uri( $stylesheet_uri, $stylesheet_dir_uri ) {
...
-		ob_end_clean();
-
-		$items .= '<li class="menu-search">';
-		$items .= '<svg version="1.1" aria-hidden="true" class="penguin-icon-search">';
-		$items .= '<use xlink:href="' . esc_url( get_template_directory_uri() ) . '/icons.svg#penguin-icon-search"></use>';
-		$items .= '</svg>' . $searchform . '</li>';
+		$items .= '<li class="menu-search">
+			<svg version="1.1" aria-hidden="true" class="penguin-icon-search">
+			<use xlink:href="' . esc_url( get_template_directory_uri() ) . '/icons.svg#penguin-icon-search"></use>
+			</svg>' . get_search_form( $echo = false ) . '</li>';
```
Comment (grappler) [inline inc/extras-penguin-gold.php]: You only need to do `get_search_form( false )`

File: searchform.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+<form role="search" method="get" class="search-form" action="<?php echo home_url( '/' ); ?>">
+	<label>
+		<span class="screen-reader-text"><?php echo _x( 'Search for:', 'label', 'penguin' ) ?></span>
+		<input type="search" class="search-field"
+			placeholder="<?php echo esc_attr_x( 'Search …', 'placeholder', 'penguin' ) ?>"
+			value="<?php echo get_search_query() ?>" name="s"
+			title="<?php echo esc_attr_x( 'Search for:', 'label', 'penguin' ) ?>" />
+	</label>
+	<input type="submit" class="search-submit screen-reader-text" value="<?php echo esc_attr_x( 'Search', 'submit button', 'penguin' ) ?>" />
+</form>
```
Comment (grappler) [inline searchform.php]: I think the `<input>` should be outside the `<label>` and the submit button should be a `<button>`. See from twenty seventeen. https://github.com/WordPress/WordPress/blob/master/wp-content/themes/twentyseventeen/searchform.php

**Comment by grappler** [comment]
There is a little bug for the footer menu. If there a sub page in the menu it will add the dropdown icon. e.g. 
image

File: searchform.php (hunk 2 above)
Comment (pixelstrolch) [inline searchform.php]: Yes, that seems reasonable. My source was https://developer.wordpress.org/reference/functions/get_search_form/

**Comment by pixelstrolch** [comment]
I couldn't find a solution for the footer menu bug. Do you have an idea to solve it?

**Comment by grappler** [comment]
You  can add `'primary' === $args['theme_location']` to the if statement.

**Comment by pixelstrolch** [comment]
If I change it to `if ( 'primary' === $args['theme_location'] && in_array( 'menu-item-has-children', $item->classes, true ) && 0 === $depth )` I'm getting a white screen.

**Comment by pixelstrolch** [comment]
Awesome 👍

**Pull request merged**